src/commonMain/kotlin/exchange.dydx.abacus/processor/wallet/account/SubaccountProcessor.kt

```diff
@@ -250,7 +253,6 @@ internal open class SubaccountProcessor(
             existing.assetPositions = assetPositionsProcessor.process(
                 payload = payload.assetPositions,
             )
-            existing.orders = null
```

Comment:
not sure why we did this, also in the non-static codepath. I get that this message contains orders so we can overwrite it, but then we miss out on filled orders in that table.

Reply:
This fix makes sense... Just need to test the scenarios of user logging out or changing the env.  I think the entire subaccount state should be nuked in those cases, so that the order list becomes empty.